lib/components/FormList.js

```diff
@@ -1,107 +1,97 @@
-                    if (comp) {
-                        const formItem = formRegistry.get(schemaItem.type);
-                        formItem.itemFunction(comp, schemaItem);
-                    }
-                });
-                return item;
-            }
         }
     },
     methods: {
         init: MLFormList$init,
-        destroy: MLFormList$destroy,
-        setTemplate: MLFormList$setTemplate,
-        setOptions: MLFormList$setOptions,
         moveItem: MLFormList$moveItem
     }
 });
 
 function handleClick (type, event) {
     const component = milo.Component.getContainingComponent(event.target);
     if (component && component.name) {
-        if (component.name === 'downBtn') {
-            const item = component.getScopeParent().item;
-            item.list.owner.moveItem(item.index, item.index + 1);
-        } else if (component.name === 'upBtn') {
-            const item = component.getScopeParent().item;
-            item.list.owner.moveItem(item.index, item.index - 1);
-        } else if (component.name === 'deleteBtn') {
-            component.getScopeParent().item.removeItem();
+        const formList = component.getScopeParentWithClass('MLFormList');
+        const formItem = component.getScopeParentWithClass('MLFormListItem');
+        if (formItem) {
+            const item = formItem.item;
+            if (component.name === 'downBtn') {
+                formList.moveItem(item.index, item.index + 1);
+            } else if (component.name === 'upBtn') {
+                formList.moveItem(item.index, item.index - 1);
+            } else if (component.name === 'deleteBtn') {
+                item.removeItem();
+            }
         }
     }
 }
 
-function prepareTemplate (label, itemContent, options) {
-    const template = document.createElement('template');
-    const itemTemplate = doT.compile(formlistitem_dot);
-    const item = itemTemplate(Object.assign({}, options, {
-        label: label || 'ITEM',
-        itemContent: itemContent.trim(),
-    }));
-    template.innerHTML = item;
-    return template.content.firstChild;
+function MLFormList$init () {
+    MLFormList.super.init.apply(this, arguments);
+    this.once('childrenbound', onChildrenBound);
+}
+
+function MLFormList$moveItem (fromIndex, toIndex) {
+    const toInsert = this.model.m.splice(fromIndex, 1);
+    if (toInsert) return this.model.m.splice(toIndex, 0, toInsert[0]);
 }
 
-function MLFormList$moveItem (from, to) {
-    this.model.splice(from, 1);
-    const toInsert = this.list.item(from).data.get();
-    return this.model.splice(to, 0, toInsert);
+function onChildrenBound () {
+    const scope = this.container.scope;
+    this.__connector = milo.minder(this.model, '->>>', scope.list.data).deferChangeMode('<<<->>>');
+    scope.addBtn && scope.addBtn.events.on('click', { subscriber: addItem, context: this });
+    this.model.m.on('*', { subscriber: function () {
```

Comment:
Looks like you could clean this up to be:

`this.model.m.on('*', { subscriber: _triggerExternalPropagation, context: this })`